docs/architecture/tls13-support.md

```diff
@@ -4,17 +4,8 @@ TLS 1.3 support
 Overview
 --------
 
-Mbed TLS provides a partial implementation of the TLS 1.3 protocol defined in
-the "Support description" section below. The TLS 1.3 support enablement
-is controlled by the MBEDTLS_SSL_PROTO_TLS1_3 configuration option.
-
-The development of the TLS 1.3 protocol is based on the TLS 1.3 prototype
-located at https://github.com/hannestschofenig/mbedtls. The prototype is
-itself based on a version of the development branch that we aim to keep as
-recent as possible (ideally the head) by merging regularly commits of the
-development branch into the prototype. The section "Prototype upstreaming
-status" below describes what remains to be upstreamed.
-
+Mbed TLS provides an implementation of the TLS 1.3. The TLS 1.3 support
+enablement is controlled by the MBEDTLS_SSL_PROTO_TLS1_3 configuration option.
```

Comment:
Preexisting: The wording here is a bit tricky to understand. Perhaps something lik:
```suggestion
may be enabled using the MBEDTLS_SSL_PROTO_TLS1_3 configuration option.
```